Makefile.win

```diff
@@ -140,9 +140,11 @@ fauxton: share\www
 .PHONY: check
 # target: check - Test everything
 check: all
+	@$(MAKE) exunit
```

Comment:
I wonder if we have any exunit tests?

Reply:
I don't know, I only made it at this part equal with our `Makefile`.

Reply:
I figured it's just a main Makefile copy and it's not a blocker for PR. More of a general comment to cleanup it eventually both for Windows and in the main Makefile eventually.